stake-pool/js/test/instructions.test.ts

```diff
@@ -252,5 +261,60 @@ describe('StakePoolProgram', () => {
       expect(res.stakeReceiver).toEqual(undefined);
       expect(res.totalRentFreeBalances).toEqual(10000);
     });
+
+    it.only('withdraw to a stake account provided', async () => {
```

Comment:
Please remove the `only` here

Reply:
removing ```only``` from here won't run this test as all other tests in this file are using ```only```.Should i remove it from every test ?

Reply:
Oh gosh, I didn't realize, sorry. We can do that in a follow up then